tableview/src/main/java/com/evrencoskun/tableview/TableView.java

```diff
@@ -902,4 +902,13 @@ protected void onRestoreInstanceState(Parcelable state) {
     public View getCornerView() {
         return getAdapter().getCornerView();
     }
+
+
+    public void selectNext() {
+        if(mColumnHeaderRecyclerViews.get(mColumnHeaderRecyclerViews.size()-1).getAdapter().getItemCount() > getSelectedColumn()+ 1) {
```

Comment:
mColumnHeaderRecyclerViews.size()

Reply:
It shouldn't. In that case there wouldn' be a table